todo_app/app.py

```diff
@@ -1,54 +1,139 @@
-import os
-
+from os import access
+from functools import wraps
+import requests
 from flask import (
     Flask,
     render_template, 
     request, 
     redirect,
     url_for,
 )
+from flask_login import (
+    LoginManager, 
+    login_required, 
+    login_user, 
+    UserMixin, 
+    current_user,
+)
+from oauthlib.oauth2 import WebApplicationClient
 
 from todo_app.config import Config
 from todo_app.data.mongodb import MongoDB, CardStatus
 from todo_app.views.views import ViewModel
 
 
+class User(UserMixin):
+    def __init__(self, name):
+        self.id = name 
+
+
+login_manager = LoginManager()
+
+# Bad practice, but hardcoded for training purposes
```

Comment:
We could always make this a bit better by extracting it to an environment variable - but agree it would want to be more dynamic eventually anyway!